Gemfile

```diff
@@ -11,7 +11,7 @@ end
 
 group :test do
   # can relax version requirement for Rails 5.2.beta3+
-  gem "minitest", "~> 5.10.3"
+  gem "minitest", "5.10.3"
```

Comment:
I could be wrong, but since 5.10.3 is the only version that matches the `~> 5.10.3` constraint, this change has no actual effect, right?

Reply:
Sorry. This was part of some flailing I did around the Minitest versions, and I shouldn't have let it slip through to the PR. :-(